roles/matrix-backup-borg/templates/config.yaml.j2

```diff
@@ -26,6 +26,16 @@ consistency:
     - archives
 
 hooks:
+{% if matrix_backup_borg_postgresql_enabled %}
```

Comment:
Adding ` and matrix_backup_borg_postgresql_databases|length > 0` here may help prevent generate an empty `postgresql_databases:` list under some weird conditions (Postgres backups enabled, but list of database is empty for whatever strange reason).